.jenkins/common_job_properties.groovy

```diff
@@ -86,14 +86,19 @@ class common_job_properties {
   // Sets the pull request build trigger.
   static def setPullRequestBuildTrigger(def context,
                                         def commitStatusContext,
-                                        def successComment = '--none--') {
+                                        def successComment = '--none--',
+                                        def postCommitPhrase = '') {
```

Comment:
It is the phrase used to trigger, but its presence or absence is what sets whether we're setting the postcommit option to trigger on a phrase and only on that phrase. I could split it out into two options: triggerPhrase and triggerOnly, which is relevant to your below comment. I do agree that for now there isn't such a case, and thus postCommitPhrase seems to me the appropriate name.